docs/pages/implementation/nuts-auth.rst

```diff
@@ -0,0 +1,225 @@
+.. _nuts-oauth-implementation:
+
+Nuts OAuth implementation manual
+################################
```

Comment:
maybe you can include/link to a diagram from the RFCs to help visualize the process?